docs/administration/snapshot-management.md

```diff
@@ -0,0 +1,40 @@
+# Snapshot Management
+
+The overall configuration of Kura is stored in an XML file called a snapshot. This file includes all of the parameters for every service running in Kura. The original configuration file is named **snapshot_0.xml**. This section describes how snapshots may be used.
+
+Each time a configuration change is made to one of the Kura components, a new XML file is created using the naming convention **snapshot_[time as a long integer].xml**. The nine most recent snapshots are saved, as well as the original snapshot 0.
+
+## How to Access Snapshots
+To display snapshots using the [Gateway Administration Console](console.html), select **Settings** from the **System** area, and then click on the **Snapshots** tab. The following three operations are available: **Download**, **Upload and Apply**, and **Rollback**.
```

Comment:
Link is broker, please use [Gateway Administration Console](../gateway-configuration/gateway-administration-console.md)